ushahidi/platform-client
# 3896 delete warning

This pull request makes the following changes:
- Adds a new notification with an extra check where the user writes the name of the survey to confirm deletion.
- Adjusts the warning text to be more clear

Testing checklist:
- [ ] Go to settings->surveys
- [ ] Click on delete
- [ ] A notification-modal should appear:
![Screenshot 2020-03-26 at 10 02 40](https://user-images.githubusercontent.com/8624777/77628871-f0699080-6f48-11ea-9bae-ea8e5e53af34.png)
- Click on Cancel
 - [ ] The modal should close
- [ ] The survey should not be deleted
- Click on delete again
- Write a text in the input-box that is not the survey-name
- Click on delete
- [ ] An error-label should appear
- [ ] The survey should not be deleted 
- Write the correct survey-name in the box
- [ ] The modal should close
- [ ] The survey should be deleted


Fixes ushahidi/platform#3896 .

Ping @ushahidi/platform
